astropy/timeseries/periodograms/lombscargle/implementations/main.py

```diff
@@ -114,7 +123,7 @@ def lombscargle(
     method="auto",
     assume_regular_frequency=False,
     normalization="standard",
-    fit_mean=True,
+    fit_mean=None,
```

Comment:
Can we leave this at `fit_mean=True`, and just raise a `DeprecationWarning` in cases where it is currently dropped? It seems strange to change the default if we're not changing behavior.

Reply:
yes, this is redundant now anyway. Thanks for pointing it out !